yarn.lock

```diff
@@ -60,6 +60,46 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@aws-crypto/cache-material@npm:^3.1.0":
```

Comment:
@mikewolfd i can't remember but I feel like dave said something at one point about not committing the yarn-lock file? Or was that package-lock?